### DISCUSSION TITLE
Diagram of Active Cases Per Province

### DISCUSSION THREAD

**Discussion by patel8786** [dsfsi/covid19za#788] [Q&A] [answered]
Hi all

On wikipedia there's a diagram from 13 December of active cases per province [https://commons.wikimedia.org/wiki/File:Za-covid-actives.svg](url)
The source references this project.

I'm trying to get an update of this diagram, does anyone know how I can get this?

Many thanks

**Comment by shaze** [comment, marked as answer]
I've just updated it with figures up those released on Friday night.

Just be careful -- the number of active cases is IMO one of the least reliable statistics
